lab1/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+
+<html>
+	<head>
+		<title>McNugget Clicker</title>
+		<link rel="stylesheet" href="app.css">
+	</head>
+	
+	<body class = "body">
+		
+		<div class = "title">
+			<h1> McNugget Clicker</h1>
+		</div>
+		
+		<div class = "story">
+			Your chicken nuggets are the talk of the town -FoxNews<br/>
+			Your chicken nuggets are sold globally -CNN<br/>
+			We are running out of chicken nuggets -McFarmer<br/>
+		</div>
+		<div class ="jj">
+		<div class = "button-buy">
+			<button><h1>McCursor:</h1>Automatically generates nuggets per second($1)</button>
```

Comment:
While I understand this may include some of the information I have in the mockup, this line misses some information I have – quantity. You will also need to display how many items user have bought this item later.